src/main/java/net/snowflake/client/core/SFLoginOutput.java

```diff
@@ -98,6 +104,14 @@ String getMfaToken() {
     return mfaToken;
   }
 
+  public String getOauthAccessToken() {
```

Comment:
new public methods may be internal?

Reply:
Removed public modifier, it was actually unnecessary